<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorLevels</class>
 <widget class="QDialog" name="ColorLevels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>色阶</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image.qrc">
    <normaloff>:/icon/color/Image/stock-color-levels.png</normaloff>:/icon/color/Image/stock-color-levels.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>通道：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_channels">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>明度</string>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/Image/stock-channel.png</normaloff>:/icon/Image/stock-channel.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>红色</string>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/Image/stock-channel-red.png</normaloff>:/icon/Image/stock-channel-red.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>绿色</string>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/Image/stock-channel-green.png</normaloff>:/icon/Image/stock-channel-green.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>蓝色</string>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/Image/stock-channel-blue.png</normaloff>:/icon/Image/stock-channel-blue.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_reset_channel">
       <property name="text">
        <string>重置通道</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>输入色阶</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_hist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>362</width>
          <height>86</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>阴影：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_low_input">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_low_input">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>中间调：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_gamma">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
          <property name="invertedControls">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_gamma">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>高光：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_high_input">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_high_input">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pushButton_black">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icon/color/Image/stock-color-picker-black.png</normaloff>:/icon/color/Image/stock-color-picker-black.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_white">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icon/color/Image/stock-color-picker-white.png</normaloff>:/icon/color/Image/stock-color-picker-white.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输出色阶</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>黑场：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_low_output">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_low_output">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>白场：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_high_output">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_high_output">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>所有通道</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_auto">
        <property name="text">
         <string>自动</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_black_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/color/Image/stock-color-picker-black.png</normaloff>:/icon/color/Image/stock-color-picker-black.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_gray">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/color/Image/stock-color-picker-gray.png</normaloff>:/icon/color/Image/stock-color-picker-gray.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_white_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Image.qrc">
          <normaloff>:/icon/color/Image/stock-color-picker-white.png</normaloff>:/icon/color/Image/stock-color-picker-white.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider_low_input</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_low_input</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_low_input</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_low_input</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_high_input</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_high_input</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_high_input</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_high_input</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_low_output</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_low_output</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_low_output</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_low_output</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_high_output</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_high_output</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_high_output</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_high_output</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>ColorLevels</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ColorLevels</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
